docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -39,13 +39,18 @@ Configuration
 
 To configure the Kafka connector, create a catalog properties file
 ``etc/catalog/kafka.properties`` with the following contents,
-replacing the properties as appropriate:
+replacing the properties as appropriate.
+
+In some cases, such as when using specialized authentication methods, it is necessary to specify
+additional Kafka client properties in order to access your Kafka cluster. To do so,
+add the ``kafka.config.resources`` property to reference your Kafka config files:
 
 .. code-block:: text
 
     connector.name=kafka
     kafka.table-names=table1,table2
     kafka.nodes=host1:port,host2:port
+    kafka.config.resources=/etc/consumer.properties
```

Comment:
If you specify extra properties for just the consumer, will these properties be used for the producer and admin factories as well?

Reply:
Yes the properties will be applied to all type of Kafka clients. Good thing about Kafka is that it ignores properties that doesn't match the client type.

Reply:
Maybe I should rename the example config name to native_kafka.properties?